packages/ra-ui-materialui/src/detail/Show.js

```diff
@@ -193,4 +193,4 @@ Show.propTypes = {
     title: PropTypes.any,
 };
 
-export default withStyles(styles)(Show);
+export default Show;
```

Comment:
It does not make much sense to have both a named and a default export for the same object. Remove the named export (you will have to change the import in Show.spec.js, too).

Reply:
sure